zoobc/zoobc-core
# Fix transaction file generated by GRPC

## Description
The previous transaction.pb.go doesn't serialize `fromBlock` and `toBlock` parameter. This PR fix the problem.

## Breakdown
- [x] fix generated file `transaction.pb.go`
- [x] fix count of result when getting transactions by block

## Reference Issue
Close #numberIssue

## Step to Test (optional)
- Run a node
- request GetTransactions with `fromBlock` and `toBlock` parameter
- make sure the results are correct
